vhost-device-console/src/main.rs

```diff
@@ -65,6 +65,10 @@ struct ConsoleArgs {
     /// `tcp_port + 1`, ... , `tcp_port + (N - 1)`.
     #[clap(short = 'p', long, value_name = "PORT", default_value = "12345")]
     tcp_port: String,
+
+    /// Specify the maximum size of vritqueue, the default is 128.
+    #[clap(short = 'q', long, default_value_t = 128)]
+    max_queue_size: usize,
```

Comment:
in `vsock` we have `queue_size`, I'd like to be consistent, but honestly I prefer more your version (just a comment, no need to change anything here, but I'm thinking if it makes sense to change the one in `vsock`)

Reply:
Of course! I'll raise another PR for vsock after this one get merged.

Reply:
Maybe I explained it wrong, vsock already allows this, but the parameter is called `--queue-size` instead of `--max-queue-size` . I'd like to be consistent, but I prefer more `--max-queue-size`, so I don't know if it makes sense to change vsock or we'll break some users.

Reply:
I see, and I perfer to be consistent as well. We can add `--max-queue-size` and mark `--queue-size` as deprecated, then remove it after a couple of releases.

Reply:
@justxuewei yep, I agree on that. If you have time, please go head and submit a PR!